src/main/java/io/cdap/plugin/zendesk/connector/ZendeskConnector.java

```diff
@@ -166,12 +172,15 @@ private Map<String, Object> getResponseMap(String baseUrl) throws IOException {
           StandardCharsets.UTF_8);
         return (Map<String, Object>) GSON.fromJson(responseAsString, Map.class);
       }
+      LOG.error("Unable to fetch the response.");
       throw new HttpResponseException(statusCode, "No response.");
     }
   }
 
-  List<String> getJsonValuesFromResponse(Map<String, Object> responseMap, ObjectType object) {
+  private List<String> getJsonValuesFromResponse(Map<String, Object> responseMap, ObjectType object) {
     List<Object> responseObjects = (List<Object>) responseMap.get(object.getResponseKey());
```

Comment:
should responseObjects be a List<Map<String, Object>> ?

Reply:
yes